modules/monitoring-exposing-custom-application-metrics-for-horizontal-pod-autoscaling.adoc

```diff
@@ -174,14 +179,20 @@ spec:
 ---
 ----
 
-. Show the Prometheus Adapter image to use:
+. List the Prometheus Adapter image:
++
+[source,terminal]
+----
+$ oc get -n openshift-monitoring deploy/prometheus-adapter -o jsonpath="{..image}"
+----
 +
+.Example output
+[source,terminal]
 ----
-$ kubectl get -n openshift-monitoring deploy/prometheus-adapter -o jsonpath="{..image}"
 quay.io/openshift-release-dev/ocp-v4.3-art-dev@sha256:76db3c86554ad7f581ba33844d6a6ebc891236f7db64f2d290c3135ba81c264c
```

Comment:
Is this procedure overall supposed to be showing what looks like a development / ART image? Or showing a 4.3 version specifically?

Reply:
@adellape Prometheus Adapter is Tech Preview in 4.6. This is stated at the top of the assembly and the whole of this assembly is subject to that. Following a discussion with @juzhao just now, I have removed the '.Example output' from step 5 so that we don't highlight the ART development image specifically. It does need to be featured in the YAML file in step 6 though and called out as being the output from step 5.

I have just tested this assembly and updated the image reference in the YAML to include the full path. I noticed that the prometheus-adapter pod in the user-defined project was in an `ImagePullBackoff` state because the full image path is required.

In a later PR, I think we need to add verification steps to this procedure. I have made a note of that for a later date.

Reply:
I have applied the changes to my local clone and will push to my fork shortly.